src/lighteval/models/base_model.py

```diff
@@ -345,6 +352,97 @@ def greedy_until_with_logits(
             override_bs=override_bs,
         )
 
+    def greedy_until_multi_turn(
```

Comment:
This function needs to exist in the parent class and raise a "not implemented exception" when people try to access it, for example using text inference models